### ISSUE TITLE
java.lang.IllegalStateException: Mis matched lengths:

### ISSUE THREAD

**Issue by WenkaiBai** [yuripourre/dl4j-house#1]
Hi, I download your code and run it. but find this error:

Exception in thread "main" java.lang.IllegalStateException: Mis matched lengths: [12288] != [4096]
	at org.nd4j.linalg.util.LinAlgExceptions.assertSameLength(LinAlgExceptions.java:39)
	at org.nd4j.linalg.api.ops.BaseTransformOp.<init>(BaseTransformOp.java:48)
	at org.nd4j.linalg.api.ops.impl.transforms.Set.<init>(Set.java:25)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.assign(BaseNDArray.java:1122)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.put(BaseNDArray.java:1837)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.putRow(BaseNDArray.java:3324)
	at org.deeplearning4j.datasets.fetchers.BaseDataFetcher.initializeCurrFromList(BaseDataFetcher.java:87)
	at com.paypal.dlj4.ImageDataFetcher.fetch(ImageDataFetcher.java:49)
	at org.deeplearning4j.datasets.iterator.BaseDatasetIterator.next(BaseDatasetIterator.java:59)
	at org.deeplearning4j.datasets.iterator.BaseDatasetIterator.next(BaseDatasetIterator.java:36)
	at com.paypal.dlj4.DLObjectDetector.trainModel(DLObjectDetector.java:156)
	at com.paypal.dlj4.DLObjectDetector.main(DLObjectDetector.java:72)

private static void trainModel(DataSetIterator iterator, MultiLayerNetwork model) {
        DataSet lfwNext;
        log.info("Train model....");
        model.setListeners(Arrays.asList((IterationListener) new ScoreIterationListener(listenerFreq)));

        while (iterator.hasNext()) {
            lfwNext = iterator.next();  // Line Number 156
            lfwNext.scale();
            trainTest = lfwNext.splitTestAndTrain(splitTrainNum, new Random(seed)); // train set that is the result
            trainInput = trainTest.getTrain(); // get feature matrix and labels for training
            testInput.add(trainTest.getTest().getFeatureMatrix());
            testLabels.add(trainTest.getTest().getLabels());
            model.fit(trainInput);
        }
    }

Do you know why this happend?

**Comment by whoami-zy** [comment]
Hello！ How do you solve this problem  thanks！

**Comment by yuripourre** [comment]
Sorry for the delayed response, this was probably because of image sizes.
I fixed the example and updated to dl4j 0.9.1.

Thank you for the messages!
